CMakeLists.txt

```diff
@@ -247,6 +247,12 @@ ELSE()
     SET (MARIADB_CLIENT_LIBS    -L${SERVER_BUILD_INCLUDE_DIR}/../libmariadb/libmariadb/ libmariadb.so)
 ENDIF()
 
+IF (INSTALL_LAYOUT)
```

Comment:
I wonder what is the semantics of INSTALL_LAYOUT variable?

Reply:
I would need to waste time studying...